### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by de-oz**
#### Summary

Turned the semicolon at the end of the first list item into a period to make it the same way as the other items of the list. Wrapped "arguments" object name in backticks as is done elsewhere on the page. Removed a period at the end of the heading.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/functions/rest_parameters/index.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ There are three main differences between rest parameters and the
   the parameters — including the parameters in the `...restParam` array —
   bundled into one array-like object.
 
-### From arguments to an array
+### From `arguments` to an array
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/functions/rest_parameters/index.md:82]: Our writing style guide forbids the use of backticks in headings.

```suggestion
### From arguments to an array
```

File: files/en-us/web/javascript/reference/functions/rest_parameters/index.md [hunk 2]
```diff
@@ -187,7 +187,7 @@ let arr = multiply(2, 15, 25, 42)
 console.log(arr)  // [30, 50, 84]
 ```
 
-### Rest parameters are real arrays; the arguments object is not.
+### Rest parameters are real arrays; the `arguments` object is not
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/functions/rest_parameters/index.md:190]:
```suggestion
### Rest parameters are real arrays; the arguments object is not
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by de-oz** [comment]
I've removed previously added backticks in the headings as suggested as well as removed them from the other heading where they were used around the same object name.

**Review by teoli2003 — APPROVED** [review]
👍  Thank you. Yes, removing the backticks is on my to-do list, but not very high. So any help there is welcome.

**Pull request merged**